### PR TITLE
feat(zoomLevel): remember zoom level

### LINKED ISSUE #320: UX: Remember Zoom level
**Problem**
On the screen that I usually use, both screens fit on the width if the zoom level is 70%. Every time that I close and open the app again, it resets back to 60%. 

**Solution**
Remember zoom level when the app is closed and opened again.

### PR DESCRIPTION
**Pull request by jzabala**
- **remember zoom level**

Screen Shot 2020-07-16 at 11 09 13 PM

- **add zoom level increment shortcut**

In Mac the shortcut is `Command` + `+`. The same shortcut for decrement already exists (`Command` + `-`).

Closes #320

### CODE REVIEW

File: desktop-app/app/reducers/browser.js [hunk 1]
```diff
@@ -237,7 +237,7 @@ export default function browser(
       ? getLastOpenedAddress()
       : getHomepage(),
     currentPageMeta: {},
-    zoomLevel: 0.6,
+    zoomLevel: _getUserPreferences().zoomLevel || 0.6,
```
Comment (manojVivek) [inline desktop-app/app/reducers/browser.js:240]: @jzabala Can you please add `zoomLevel` field to `UserPreferenceType` type above?

Comment (jzabala) [reply desktop-app/app/reducers/browser.js:241]: Added!

**Comment by jzabala** [comment]
> Thank you so much for fixing this, just one minor feedback and it is good to be merged. 🙌

Thank you for taking the time to review the PR manojVivek.

I solved all the requested changes.

**Comment by manojVivek** [comment]
@jzabala Great, looks good. Merging it now. Looking forward to more collaboration in the future. 🙌

**Comment by manojVivek** [comment]
@all-contributors Please add @jzabala for code.

**Comment by manojVivek** [comment]
@jzabala  Two things:

1. Can you let us know your Twitter handle? We would love to give a shout out for your work from our twitter account. 
2. Please join our community on Gitter - https://gitter.im/responsively-app/, for any discussions in the future.

**Comment by jzabala** [comment]
> 1. Can you let us know your Twitter handle? We would love to give a shout out for your work from our twitter account.

@jzabalasanchez

> 2. Please join our community on Gitter - https://gitter.im/responsively-app/, for any discussions in the future.

Sure 🙂

**Pull request merged**